vm/src/sector/registered_proof.rs

```diff
@@ -11,13 +11,25 @@ use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 #[repr(u8)]
 pub enum RegisteredProof {
     StackedDRG32GiBSeal = 1,
-    StackedDRG32GiBPoSt = 2,
+    StackedDRG32GiBPoSt = 2, // unused (revisit if being removed)
```

Comment:
Maybe add "TODO" to the comment. Makes it easier to search for all TODOs when we need to